### ISSUE TITLE
[Small] Use `Environment.NewLine` which is Platform dependant

### ISSUE THREAD

**Issue by pjmagee** [wingyplus/dagger-dotnet-sdk#15]
This is the appropriate way to handle the concept of `\n` or `\n\r`

https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Environment.cs,63a04833d43dd9d3

**Comment by pjmagee** [comment]
Well, I ran into a fun issue running the Code on Windows, but the NewLine from container was \n So it didnt work -.- Maybe this idea of mine isn't as good as I thought

**Comment by wingyplus** [comment]
Is this issue still occurred? Do you have any reproduce code?

**Comment by pjmagee** [comment]
> Is this issue still occurred? Do you have any reproduce code?

Yeah, the problem is, if you are running code on windows, it will \r\n, but the output was supposed to be \r, because its running in containers with linux. So issue is using `Environment.NewLine` in one case, but the runtime is different, it can cause issues depending on environment code was executed on. 

I forgot the specific example :(
